managers/oclcClassify.py

```diff
@@ -67,6 +67,27 @@ def getClassifyResponse(self):
 
         return self.parseXMLResponse()
 
+    def generate_identifier_query(self):
+        identifier_map = { "isbn": "bn",
+                          "issn": "in",
+                          "oclc": "no"}
```

Comment:
We could add other identifiers here, like lccn, which we store but don't currently use (we would need to add this 
"upstream" as well). However, I don't see a reason to do this unless we think it would improve the clustering process.